### PR TITLE
Fine tune Monitoring guide (big)

### PR DESCRIPTION
**Pull request by Lennonka**
Polishing the new Monitoring guide.
**Recommendation for reviewers**: Check out the preview for smooth reading experience.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.9 (Satellite 6.15) if available
* That's all.

### CODE REVIEW

File: guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc
Comment (Lennonka) [inline guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc:16]: We generally try to avoid screenshots. Can I have an ack from peers?

File: guides/common/attributes-titles.adoc
Comment (evgeni) [inline guides/common/attributes-titles.adoc:21]: This should also be updated in `web/content/js/versions.js`

**Review by evgeni — CHANGES_REQUESTED** [review]
I could not spot any technical changes, so ACK on that side :)
"Request changes" just because the title was not updated in `versions.js`

File: guides/common/modules/proc_retrieving-archived-metrics-using-cli.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_retrieving-archived-metrics-using-cli.adoc:27]:
```suggestion
* To list disk write operations per partition, with a 2-second interval, over the time period between 14:00 and 14:15:
```

File: guides/common/modules/proc_retrieving-live-metrics-using-cli.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_retrieving-live-metrics-using-cli.adoc:2]: Imho "using" was better here, but should be "using the"

File: guides/common/modules/proc_retrieving-live-metrics-using-cli.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_retrieving-live-metrics-using-cli.adoc:21]: Since this is example output, it can stay non-italicized, but I don't care either way.

File: guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc:4]:
```suggestion
You can view the Grafana dashboard which provides default widgets displaying the values of metrics.
```

File: guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc
Comment (adamlazik1) [reply guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc:16]: Yeah.

File: guides/common/modules/proc_verifying-pcp-configuration.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_verifying-pcp-configuration.adoc:11]:
```suggestion
Example output of the `pcp` command:
```

**Review by adamlazik1 — COMMENTED** [review]
Looks good overall. Pointed out some minor things.

File: guides/common/attributes-titles.adoc
Comment (Lennonka) [reply guides/common/attributes-titles.adoc:21]: Thank you!

File: guides/common/modules/proc_retrieving-live-metrics-using-cli.adoc
Comment (Lennonka) [reply guides/common/modules/proc_retrieving-live-metrics-using-cli.adoc:2]: It actually should be "by using the CLI" for complete clarity, which is so damn long :D I was thinking about leaving it out altogether. What do you think about that?

File: guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc
Comment (Lennonka) [reply guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc:4]: A comma before "which" should be correct, because it introduces a nonrestrictive clause.
https://www.merriam-webster.com/grammar/when-to-use-that-and-which

File: guides/common/modules/proc_verifying-pcp-configuration.adoc
Comment (Lennonka) [inline guides/common/modules/proc_verifying-pcp-configuration.adoc:28]: @evgeni FYI, this is a bit confusing, because `pmwebd` is not in the example output above. Are we missing a piece of the output? If not, then do we have to mention this daemon, when it isn't in the example?

Comment (evgeni) [reply guides/common/modules/proc_verifying-pcp-configuration.adoc:28]: Huh, this one did slip in. Yeah, pmwebd is not a thing anymore in PCP5+, let me see if we need to mention proxy or nothing.

Comment (evgeni) [reply guides/common/modules/proc_verifying-pcp-configuration.adoc:28]: Yeah, it should be the proxy. So I guess we should write "Proxy for Performance Metrics Collector Daemon (pmproxy)" or "Performance Metrics Proxy Daemon (pmproxy)" or something similar (that wording is taken from their own docs: https://man7.org/linux/man-pages/man1/pmproxy.1.html https://pcp.readthedocs.io/en/latest/QG/QuickReferenceGuide.html#pcp-web-services )

File: guides/common/modules/proc_retrieving-live-metrics-using-cli.adoc
Comment (adamlazik1) [reply guides/common/modules/proc_retrieving-live-metrics-using-cli.adoc:2]: Yeah that works.

File: guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc
Comment (adamlazik1) [reply guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc:4]: I see :+1:

File: guides/common/modules/proc_verifying-pcp-configuration.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_verifying-pcp-configuration.adoc:11]:
```suggestion

Example output of the `pcp` command:
```

**Review by adamlazik1 — APPROVED** [review]
LGTM stylewise. One last nitpick.

File: guides/common/modules/proc_verifying-pcp-configuration.adoc
Comment (Lennonka) [reply guides/common/modules/proc_verifying-pcp-configuration.adoc:28]: Perfect, thanks!

File: guides/common/modules/proc_verifying-pcp-configuration.adoc
Comment (Lennonka) [reply guides/common/modules/proc_verifying-pcp-configuration.adoc:13]: @adamlazik1 I did it a bit differently, let me know if it works.

File: guides/common/modules/con_metrics-overview.adoc
Comment (maximiliankolb) [inline guides/common/modules/con_metrics-overview.adoc:4]: What does "solution" mean? Is there a more descriptive word? Or maybe "... set up a third party solution to ..."? or "external solution"?

File: guides/common/modules/proc_changing-default-logging-interval.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_changing-default-logging-interval.adoc:5]: means > results in?

File: guides/common/modules/proc_retrieving-archived-metrics.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_retrieving-archived-metrics.adoc:6]: Maybe reword to "Examples"? I like that you went with an unordered list here.

File: guides/common/modules/proc_viewing-data-storage-statistics.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_viewing-data-storage-statistics.adoc:13]: "... on your {ProjectServer}"?

File: guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc:4]: this sounds off to me. Maybe "... view the Grafana dashboard to view widgets ..."?

**Review by maximiliankolb — COMMENTED** [review]
Thanks Lena! Left some suggestions; none of them are blocking this PR. I cannot comment too much on the actual content though.

File: guides/common/modules/proc_verifying-pcp-configuration.adoc
Comment (adamlazik1) [reply guides/common/modules/proc_verifying-pcp-configuration.adoc:13]: That works, thanks!

File: guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc
Comment (Lennonka) [reply guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc:4]: How about: "... view the Grafana dashboard, which displays the values of metrics as widgets." Would that work better?

**Pull request merged**
